src/test/integration/IntegrationChecks.t.sol

```diff
@@ -558,14 +598,16 @@ contract IntegrationCheckUtils is IntegrationBase {
         Withdrawal memory withdrawal,
         IStrategy[] memory strategies,
         uint[] memory withdrawableShares
-    ) internal {
+    ) public {
         /// Complete withdrawal(s):
         // The staker will complete the withdrawal as shares
         //
         // ... check that the withdrawal is not pending, that the token balances of the staker and operator are unchanged,
         //     that the withdrawer received the expected shares, and that that the total shares of each o
         //     strategy withdrawn remains unchanged
-        assert_WithdrawalNotPending(delegationManager.calculateWithdrawalRoot(withdrawal), "staker withdrawal should no longer be pending");
+        assert_WithdrawalNotPending(
+            delegationManager().calculateWithdrawalRoot(withdrawal), "staker withdrawal should no longer be pending"
```

Comment:
Followup work - we should be able to only pass in `withdrawal` and have the underlying method get the root.